src/Core/Framework/DependencyInjection/Configuration.php

```diff
@@ -181,6 +181,9 @@ private function createAdminWorkerSection(): ArrayNodeDefinition
                 ->booleanNode('enable_admin_worker')
                     ->defaultValue(true)
                 ->end()
+                ->scalarNode('memory_limit')
+                    ->defaultValue('128M')
```

Comment:
is there any reason why its soo low?

Reply:
Yes, this the default value you guys recommend for the cli worker: https://developer.shopware.com/docs/guides/hosting/infrastructure/message-queue#cli-worker

Generally speaking keeping this value low is helpful to us because that means a single message can be certain that there is enough memory to execute its logic. This memory limit only ensures that the worker is restarted. A single message still has the full php memory limit available which are 512MB according to you system requirements. After the message has been processed the worker will check if more than 128MB of memory is used and stop the execution of further messages.

Stopping the admin worker will return to the administration that messages have been processed and the administration will restart the admin worker right away to ensure any further messages can be processed. That way all messages have enough head room to execute before running into the php memory limit.